Scribe/Resources/Localizable.xcstrings

```diff
@@ -0,0 +1,157 @@
+{
```

Comment:
Really appreciate the focus on L10n :) We might end up switching this over to a JSON as the general idea is that we'll be linking it to the Android repo as well (see [Scribe-i18n#1](https://github.com/scribe-org/Scribe-i18n/issues/1)), but for now this is great and sets us up perfectly for the later switch!

Reply:
Hmm good point 🤔 At least the call sites can be found pretty easily 😄

Reply:
Made #392 to cover this :)